docs/clusterfuzz/architecture.md

```diff
@@ -21,35 +22,41 @@ triaging crashes, minimizing reproducers, [bisecting], and verification of fixes
 ClusterFuzz is written in Python and Go. It runs on **Linux**, **macOS**, and **Windows**.
 
 ## Requirements
-It runs on the [Google Cloud Platform](https://cloud.google.com/), and depends
+ClusterFuzz runs on the [Google Cloud Platform](https://cloud.google.com/) and depends
 on a number of services:
-- Compute Engine (Not strictly necessary. Bots can run anywhere).
+- Compute Engine (Not strictly necessary. Bots can run anywhere.)
 - App Engine
 - Cloud Storage
 - Cloud Datastore
 - Cloud Pub/Sub
 - BigQuery
 - Stackdriver Logging and Monitoring
 
-**Note**: The only bug tracker supported now is the Chromium hosted
+**Note**: The only bug tracker we currently support is the Chromium-hosted
 [Monorail](https://opensource.google.com/projects/monorail). Support for custom
-bug trackers will be added in the near future.
+bug trackers will be added in the future.
```

Comment:
(Kevin, please ignore) @oliverchang do you want to re-write this sentence to mention bug tracker extendability?

Reply:
Sure, added to my TODO.